egs/librispeech/asr1/conf/tuning/lm_transformer.yaml

```diff
@@ -0,0 +1,20 @@
+# see https://arxiv.org/pdf/1905.04226.pdf
```

Comment:
Can you make it more structured by separating the topology definition, optimization related etc, similar to the e2e ASR config?

Reply:
I like adding comments in header.
(Intension of configuration, requirements, how long does it take....)
If you add brief comment, that is nice.

Reply:
I see. I will make sections with spaces and add more comments for choices and requirements.